src/view/treeNodes/repositoryChangesNode.ts

```diff
@@ -61,6 +63,7 @@ export class RepositoryChangesNode extends DescriptionNode implements vscode.Tre
 	}
 
 	async getChildren(): Promise<TreeNode[]> {
+		await this._reviewManager.refreshSinceReview;
```

Comment:
Ideally, now that the other dependency on `ReviewManager` is gone, we'd be able to remove this one too. I do see that this makes showing progress work easily though. Let's save it for a separate PR during debt week and find a way to add something to `ReviewModel`.

Reply:
I'll remove it, but will have to figure out another way to show progress.